GOVERNANCE.md

```diff
@@ -18,15 +18,61 @@ This project may include the following roles. Additional roles may be adopted an
+
+**Calling a vote.** While "calling a vote" seems like a slow and bureaucratic process, its goal is to ensure that all stakeholders had a say in the process of making a decision. In an effort to keep things simple and lean, this step can be considered "integrated" into the previous (informal) step of Informal Consensus Seeking if there's enough support and agreement so that is clear that the proposal has the support from the majority of `maintainers`.
+
+If that's not the case and two or more parties are not aligned, any `maintainer` can call the matter to a vote. When that member (the sponsor) calls the vote, they shall summarize the proposal in its current to make sure the proposal is clear to everyone. After the proposal is seconded by at least another `maintainer`, voters have 4 days to vote. `Maintainers` may consider longer voting periods as necessary for special circumstances, or shorter periods only if all voting members are present. The decision will be determined by a simple majority of non-blank votes for binary decisions (i.e., approving a proposal) and ranked choice for multi-class decisions (one among many, or several among many). The sponsor may update the proposal at any point during the voting period, in which case the voting period will be reset.
+
+NOTE: While only `maintainers` can vote, anyone can comment and participate to the discussion around the proposal, in the spiriti of opennes and inclusivity at the base of this project.
+
+**Voter participation and quorum.** There is no minimum quorum for a vote to be considered valid, unless a voter had expressed their inability to vote and asked for an extended voting time window. All votes must include a "abstinent" option, for voters to express their desire to be neutral to decision while still casting a vote.
+
+All `maintainers` are highly encouraged to participate and exercise their rights expressing their opinion in at least 2/3 of formal votes per calendar year. Members that have not met the 2/3 vote participation threshold for a year and have a low contribution rate in that year may be to step down from their `maintainer` status at the end of that year. Those individuals remain eligible to rejoin the `maintainers` group in the future as they become available to participate at the required level. The quorum for all formal votes will be 50% and a “blank” option will always be included, with the “blank” option counting towards the quorum but not included in totals for calculating results.
+
+**Recording.** Once a decision has been made during the consensus-seeking phase or by a formal vote, the initiative `sponsor` will record the decision in the related discussion, for posterity.
+
+### 3.2.2. Things to avoid
```

Comment:
```suggestion
### 3.2.2. Keys to Successful Consensus
```

A few of the things in this section are "don't do this" and a few are "remember to do this", so I dropped in some proposed language making them all things _to do_. Just a style thing, feel free to take or leave.

Reply:
let's be affermative! :)